eahub/templates/eahub/group.html

```diff
@@ -54,5 +54,44 @@ <h1>{{ group.name }}</h1>
       {% include 'eahub/maps/profile_page_map.html' with lat=group.lat|default_if_none:0 lon=group.lon|default_if_none:0 %}
     </div>
   </div>
+
+  <div class="claim_group_container">
+  {% if user.is_authenticated %}
+  {% if messages %}
```

Comment:
This should be in base.html, I think. We want messages to be supported everywhere.